main.py

```diff
@@ -51,9 +54,14 @@
 
   # Setup
   args = parser.parse_args()
+  os.system('mkdir -p results')
```

Comment:
Please use the following to check if the dir exists and make it one go otherwise:
```py
if not os.path.exists(dir):
    os.makedirs(dir)
```